server/package.json

```
@@ -34,6 +34,6 @@
     "passport-google-oauth20": "^2.0.0",
     "passport-local": "^1.0.0",
     "passport-local-mongoose": "^6.1.0",
-    "qs": "^6.10.1"
+    "qs": "^6.10.1",
```

Essa vírgula tá quebrando o arquivo.
```suggestion
    "qs": "^6.10.1"
```